Source/WebCore/rendering/RenderText.cpp

```diff
@@ -1329,6 +1329,55 @@ UChar RenderText::previousCharacter() const
     return previousString[previousString.length() - 1];
 }
 
+// https://www.w3.org/TR/css-text-3/#small-kana
+static String convertToFullSizeKana(const String& string)
+{
+    static const uint16_t smallKanas[] = {
```

Comment:
Might use `static constexpr std::pair<uint16_t, uint16_t> mapping[]` and converted to `SortedArrayMap`.

Reply:
This feature is so rare that we might prefer optimising compilation time instead of run time for this.

Reply:
I think I disagree. Some pages might have this feature enabled gazillions of times on gazillions of content, and we want such pages to work well too.

Reply:
It's true, and they will work properly indeed, with or without a `constexpr`.

Reply:
work _well_ => with good performance

Reply:
Changed to `constexpr SortedArrayMap`